redfish-client/etc/redfish-client.conf

```diff
@@ -0,0 +1,2 @@
+[redfish-client]
+templates_path = /usr/share/redfish-client/templates
```

Comment:
I think I can replace /usr/share/redfish-client/templates with a "keyword" and do some kind of substitution. What "keyword" do you propose, so we can use the same ?
Ex : templates_path = <TEMPLATE_PATH> 
--> Where <TEMPLATE_PATH> = /usr/local/share/redfish-client/templates